src/dashboard/src/components/administration/views.py

```diff
@@ -351,6 +352,23 @@ def processing(request):
     return processing_views.index(request)
 
 
+def handle_config(request):
+    """Display or save the Handle configuration form, which allows for the
+    specification of configuration values for Handle PID creation and binding
+    using the ``bindpid`` module. State is stored in DashboardSettings table.
+    """
+    if request.method == 'POST':
+        form = HandleForm(request.POST)
+        if form.is_valid():
+            models.DashboardSetting.objects.set_dict(
+                'handle', form.cleaned_data)
+            messages.info(request, _('Saved.'))
+    else:
+        form = HandleForm(
+            initial=models.DashboardSetting.objects.get_dict('handle'))
+    return render(request, 'administration/handle_config.html', locals())
```

Comment:
I agree it's preferable to name what we are passing to the template (explicit is better than implicit) - it does make it easier to see what's going on and provides a more clear interface between the view and template. Perhaps we could adopt this more going forwards.